Doc: Improved Troubleshooting section on README (#291)
okfn-brasil/querido-diario: README.md

## Troubleshooting

### Non-specific errors when building or running

> Several error is because your user don't is from docker group. To resolve this,

Reviewer: "don't is from" -> "is not in the"?

Author: I'll fix. Txs.

> Several error is because your user don't is from docker group. To resolve this,

Reviewer 2: ```suggestion
Several error is because your user is not in the docker group. To resolve this,
```

execute the follow command and do logout and login:

```console
$ sudo usermod -aG docker $USER

> ```

Reviewer 2: Add a sentence to remember the users that they need to logout and login in the session

Author: The previous line is:

`execute the follow command and do logout and login`

It is not enough?
